src/components/search/search.spec.js

```diff
@@ -531,6 +531,38 @@ describe("Search", () => {
     });
   });
 
+  describe("tab index should be set on search button", () => {
+    it("when input field is not empty", () => {
+      wrapper = renderWrapper(
+        {
+          defaultValue: "Bar",
+          id: "Search",
+          name: "Search",
+          searchButton: true,
+        },
+        mount
+      );
+      const searchButton = wrapper.find(Button);
+      expect(searchButton.prop("tabIndex")).toEqual(0);
+    });
+
+    it("when input field has focus and is empty", () => {
```

Comment:
This should be in a describe block saying `tab index should **not** be set on search button` or similar. Or restructure these however you want as long as the describe/it descriptions will read right

Reply:
That makes sense. Cheers for spotting this. It made sense to me doing it this way but that's because I'm the one the work 😅 
I'll get it sorted now 👍🏻